test/TestPrisonersDilemma.js

```diff
@@ -175,4 +175,37 @@ contract('PrisonersDilemma', async (accounts) => {
         //Test if a player can continue to play after there is a winner
         await expectThrow(instance.playerChoose(CHOICES["Take"], { from: accounts[0] }))
     });
+
+    it("Gas Analysis", async() => {
+        var estimate = 0;
+        var receipt = 0;
+
+        console.log(web3.version.api);
+        console.log(web3.version.ethereum);
+
+        instance = await PrisonersDilemma.new(
+            [accounts[0], CHOICES["No_Choice"], 0], 
+            [accounts[1], CHOICES["No_Choice"], 0],
+            [1, 1, 1]);
+
```

Comment:
I still need to figure out a way to evaluate gas usage of initializing a contract.
Check for two things:
- wrap instantiation of a contract in an estimate gas method
- see if method exists internal to the contract similar to estimateGas() for a method

Reply:
contract estimate: https://ethereum.stackexchange.com/questions/36270/how-can-i-get-the-gas-cost-of-contract-creation-within-truffle-migrations-tes?noredirect=1&lq=1

Reply:
https://ethereum.stackexchange.com/questions/12841/with-web3-how-would-i-get-transaction-and-function-result